ci/fuzz/fuzz.cc

```diff
@@ -0,0 +1,62 @@
+#include <cassert>
+#include <cstddef>
+#include <cstdint>
+#include <cstdlib>
+#include <iostream>
+#include <optional>
+
+#include <hilti/rt/exception.h>
+#include <hilti/rt/init.h>
+#include <hilti/rt/types/reference.h>
+#include <hilti/rt/types/stream.h>
+
+#include <spicy/rt/init.h>
+#include <spicy/rt/parser.h>
+
+std::optional<std::string> name;
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size) {
+    static const spicy::rt::Parser* parser = nullptr;
+
+    if ( ! parser ) {
+        hilti::rt::init();
+        spicy::rt::init();
+
+        for ( auto* p : spicy::rt::parsers() ) {
+            parser = p;
+            if ( name && p->name == *name )
+                break;
+        }
+    }
+
+    assert(parser);
+    assert(parser->parse1);
+
+    hilti::rt::ValueReference<hilti::rt::Stream> stream;
+    stream->append(reinterpret_cast<const char*>(Data), Size);
```

Comment:
Extension for later: we could also test parsing the data incrementally, one byte at a time. But let's get this going for a while first to see if anything shows up this way.